blob/service_test.go

```diff
@@ -0,0 +1,229 @@
+package blob
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"sort"
+	"testing"
+	"time"
+
+	ds "github.com/ipfs/go-datastore"
+	ds_sync "github.com/ipfs/go-datastore/sync"
+	mdutils "github.com/ipfs/go-merkledag/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/celestia-app/pkg/shares"
+	"github.com/celestiaorg/go-header/store"
+	"github.com/celestiaorg/nmt/namespace"
+
+	"github.com/celestiaorg/celestia-node/header"
+	"github.com/celestiaorg/celestia-node/header/headertest"
+	"github.com/celestiaorg/celestia-node/share"
+	"github.com/celestiaorg/celestia-node/share/getters"
+)
+
+func TestService_GetSingleBlob(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	t.Cleanup(cancel)
+
+	blobs := generateBlob(t, []int{10, 6}, false)
+	service := createService(ctx, t, blobs)
+
+	newBlob, err := service.Get(ctx, 1, blobs[1].NamespaceID(), blobs[1].Commitment())
+	require.NoError(t, err)
+	require.Equal(t, blobs[1].Commitment(), newBlob.Commitment())
+}
+
+func TestService_GetSingleBlobInHeaderWithSingleNID(t *testing.T) {
```

Comment:
pls add some docs to these tests so they're a bit more descriptive about what's going on and what's the expected outcome.

Reply:
N/A